docs/source/programming.md

```diff
@@ -0,0 +1,223 @@
+
+- If the selected Channel is the `Microwave` channel, the system is initialized in $\left|00...0\right>$ and the interaction Hamiltonian is the [XY Hamiltonian](programming.md#xy-hamiltonian)
+
+$$H^\text{int}_{ij} =\frac{C_3}{R_{ij}^3}|1\rangle\langle 0|_i |0\rangle\langle 1|_j + |0\rangle\langle 1|_i |1\rangle\langle 0|_j$$
+
+:::{important}
+At this stage, the [interaction Hamiltonian](programming.md#22-interaction-hamiltonian) is fully determined.
+:::
+
+A `Channel` is also determined by its addressing. This defines the number of atoms that are going to be targeted by a pulse. If the addressing of a `Channel` is `Global`, all the atoms will experience the same pulse targetting the same transition. In the [Hamiltonian $H$](programming.md#2-hamiltonian-evolves-the-state), all the driving Hamiltonians $H^D_i$ are expressed as
+
+$$
+H^D_i(t) / \hbar = \frac{\Omega(t)}{2} e^{-j\phi(t)} |a\rangle_i \langle b|_i + \frac{\Omega(t)}{2} e^{j\phi(t)} |b\rangle_i\langle a|_i - \delta(t) |b\rangle_i\langle b|_i
+$$
+
+If the addressing of a `Channel` is `Local`, then only certain atoms (the atoms "target") will experience the pulse and have their evolution follow $H^D_i$. The driving hamiltonian of the other atoms is $H^D_i = \hat{0}_i$. The hamiltonian $H$ can also be rewritten:
+
+$$
+H(t) = \sum_{i \in targeted\_ atoms} H^D_i(t) + \sum_i \sum_{j<i}H^\text{int}_{ij},
+$$
+
+:::{important}
+The addressing of a `Channel` defines the number of atoms whose transition will be targeted by a pulse.
+
+The most common addressing for a `Channel` is the `Global` one: all the atoms evolve under the same [driving Hamiltonian](programming.md#21-driving-hamiltonian). 
+:::
+
+### 4. Add the Pulses
+
+By adding pulses to a channel, we incrementally construct the [driving Hamiltonian](programming.md#21-driving-hamiltonian):
+- Each `Pulse` defines the Rabi frequency $\Omega(t)$, the detuning $\delta(t)$ and the phase $\phi$ of the driving Hamiltonian over a duration $\Delta t$. Similarly, a delay sets all these parameters to zero for a given amount of time.
+- The channel dictates the states $\left|a\right>$ and $\left|b\right>$ of the driving Hamiltonian.
+
+By applying a series of pulses and delays, one defines the entire driving Hamiltonian of each atom over time.
+
+## Conclusion
+
+We have successfully defined the [Hamiltonian](programming.md#2-hamiltonian-evolves-the-state) $H$ describing the evolution of the system over time, by:
+- picking a `Device`, that defined the value of the $C_6$ or $C_3$ coefficients.
+- creating a `Register` of atoms, that defined the number of atoms used and the distance between the atoms $R_{ij}$.
+- Selecting the `Channels` of the `Device` to use, that defined the energy levels of the atoms to use: that step completely defined the [interaction Hamiltonian](programming.md#22-interaction-hamiltonian). The addressing property of each `Channel` also dictates the atoms that will be targeted by the `Pulse`.
```

Comment:
nit: There should be no comma before "that". I think "which" is more approriate here
```suggestion
- Picking a `Device`, which defines the value of the $C_6$ or $C_3$ coefficients.
- Creating a `Register` of atoms, which defines the number of atoms used and the distance between them, $R_{ij}$.
- Selecting the `Channels` of the `Device` to use, which define the energy levels of the atoms to use - thos step completely defined the [interaction Hamiltonian](programming.md#22-interaction-hamiltonian). The addressing property of each `Channel` also dictates the atoms that will be targeted by the `Pulse`.
```